examples/individual_properties/main.rs

```diff
@@ -54,7 +59,7 @@ async fn get_name_for_sensor(device: &AstarteDeviceSdk, sensor_n: i32) -> Result
 }
 
 #[tokio::main]
-async fn main() -> Result<(), AstarteError> {
+async fn main() -> Result<(), Box<dyn StdError>> {
```

Comment:
Can you use the `astarte_device_sdk::error::Error`?
```suggestion
async fn main() -> Result<(), astarte_device_sdk::error::Error> {
```

Reply:
No because there are multiple different errors. We should use a library like Anyhow here, but an easier way without dependencies is to use Box<dyn StdError>